### DISCUSSION TITLE
How to add description to commit

### DISCUSSION THREAD

**Discussion by itzyashh** [community/community#31667] [Other Feature Feedback, Questions, & Ideas]
Hello I have been using GitHub Desktop application then I switched to terminal based git , I know that you can commit using `git commit` cmd which has -m flag for message but in Github Desktop There are two options Summary( -m ig ) and Description. I want to add description to my commit, How can I do that using terminal something like -d works ?
![image](https://user-images.githubusercontent.com/82391577/188025875-cdcbed30-400d-4e56-9828-64a09893a462.png)

**Comment by toufiqul-hasan** [comment]
try this:

git commit -m "Title" -m "Description";

**Comment by airtower-luna** [comment]
If you don't specify `-m` (or one of the other options that set a commit message) `git commit` should open an editor where you can enter a commit message. The first line will be the summary, the rest what GitHub Desktop calls the description. Make sure to leave a blank line after the summary.

And yes, multiple `-m` options are also valid.

**Comment by SharfShiplo** [comment]
Run git commit without a message and it'll open up your favorite* text editor to write a commit message. Details on the format of the message will appear in the sample.

First line is the short description. Leave a blank line after it. Everything else is the extended description.

Or  There is also another straight and more clear way: 
```git
git commit -m "Title" -m "Description ..........";
```

**Reply by airtower-luna** [reply]
> *: for some values of "favorite"; results may vary

:grin: If the editor that opens isn't actually your favorite, you can [set `core.editor` with `git config`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreeditor) to the command that'll launch your actual favorite.
